testing/sync_integration_tests/BUILD.bazel

```diff
@@ -7,7 +7,7 @@ kubectl_patch(
     name = "sync_integration_tests",
     namespace = project.namespace,
     resource_type = "qjob",
-    resource_name = "kubecf-sync-integration-tests",
+    resource_name = "kubecf-sync_integration_tests",
```

Comment:
This is wrong. This is the k8s resource name, and k8s doesn't allow `_`. In fact, cf-operator converts any `_` to `-`.

Reply:
@f0rmiga what about the "-" after the "kubecf" word?